### PR TITLE
[jellyfin] convert to common library

### PR DESCRIPTION
**Pull request by onedr0p**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/jellyfin/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (bjw-s) [inline charts/jellyfin/.helmignore:1]: We should probably standardize this file across the charts somehow... this one is missing the `.vscode` IDE folder

File: charts/jellyfin/Chart.yaml [hunk 2]
```diff
@@ -1,16 +1,21 @@
 apiVersion: v2
-appVersion: 10.6.3
-description: Jellyfin is a Free Software Media System
+appVersion: 10.6.4
+description: NZBGet is a Usenet downloader client
```
Comment (bjw-s) [inline charts/jellyfin/Chart.yaml:3]: Copy/pasted from NZBGet chart :-)

File: charts/jellyfin/README.md [hunk 3]
```diff
@@ -17,6 +17,11 @@ To install the chart with the release name `my-release`:
 helm install --name my-release k8s-at-home/jellyfin
 ```
 
+The default login details (change ASAP) are:
+
+* login:jellyfin
```
Comment (bjw-s) [inline charts/jellyfin/README.md:22]: Copy/paste from nzbget?

File: charts/jellyfin/Chart.yaml (hunk 2 above)
Comment (onedr0p) [reply charts/jellyfin/Chart.yaml:3]: damn copypasata

File: charts/jellyfin/README.md [hunk 4]
```diff
@@ -28,82 +33,50 @@ helm delete my-release --purge
...
     k8s-at-home/jellyfin
 ```
-
-Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
-
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the
+chart. For example,
 ```console
-helm install --name my-release -f values.yaml stable/jellyfin
+helm install radarr k8s-at-home/jellyfin --values values.yaml
```
Comment (bjw-s) [inline charts/jellyfin/README.md:49]: Radarr reference

File: charts/jellyfin/.helmignore (hunk 1 above)
Comment (onedr0p) [reply charts/jellyfin/.helmignore:1]: I agree, my copy pasta is failing me yet again

**Review by bjw-s — APPROVED** [review]
👍

**Pull request merged**